packages/lesswrong/lib/collections/userTagRels/collection.ts

```diff
@@ -0,0 +1,58 @@
+import { getDefaultMutations, getDefaultResolvers, schemaDefaultValue } from "../../collectionUtils";
+import { foreignKeyField } from "../../utils/schemaUtils";
+import { createCollection } from '../../vulcan-lib';
+
+const schema: SchemaType<DbUserTagRel> = {
+  tagId: {
+    ...foreignKeyField({
+      idFieldName: "tagId",
+      resolverName: "tag",
+      collectionName: "Tags",
+      type: "Tag",
+    }),
+    canRead: ['guests'],
+    canCreate: ['members'],
+  },
+  userId: {
+    ...foreignKeyField({
+      idFieldName: "postId",
+      resolverName: "post",
+      collectionName: "Posts",
+      type: "Post",
```

Comment:
I assume this should actually be a user, not a post?

Fortunately it doesn't matter because Mongo doesn't actually enforce its schema…

Reply:
Oops 😬, will update this